schemas/expconf/v0/searcher-adaptive-asha.json

```diff
@@ -33,6 +33,20 @@
             "default": null,
             "minimum": 1
         },
+        "time_metric": {
```

Comment:
should time_metric and max_time be eventuallyRequired?

IIUC we won't accept asha jobs without them.

Reply:
hm... actually I'm not quite sure how to handle this one.

I suppose, since we still haven't eradicated all of the legacy expconf parsing (which is why we still allow the EOL searchers in the json-schema in the first place), the safest place for a time_metric and max_time check is actually in:

    pkg/scheams/expconf/searcher_config.go::SearcherConfig.AssertCurrent().

That will prevent dumb bugs like "`det e set-checkpoint-gc-policy` fails for old asha experiments" from happening, or either of the other two or three weird places where we wrongly try to parse an experiment config.